docs/source/index.rst

```diff
@@ -55,7 +55,7 @@ mwclient.
 Reference guide
 ---------------
 
-If you are looking for information on a specific function, class or method,
```

Comment:
oh boy, we're having opinions about the Oxford comma now?! this could get controversial. :D

Reply:
I reverted this. Honestly, I don't have a strong opinion either way and just accepted the suggestion from my spellchecker 😅